api/agent/server.go

```diff
@@ -588,15 +588,23 @@ func (s *Server) GetRouter() http.Handler {
 
 	// Tornjak specific
 	apiRtr.HandleFunc("/api/tornjak/serverinfo", s.tornjakGetServerInfo)
+	apiRtr.HandleFunc("/api/tornjak/v1/serverinfo", s.tornjakGetServerInfo)
 	// Agents Selectors
 	apiRtr.HandleFunc("/api/tornjak/selectors/register", s.tornjakPluginDefine)
+	apiRtr.HandleFunc("/api/tornjak/v1/selectors/register", s.tornjakPluginDefine)
 	apiRtr.HandleFunc("/api/tornjak/selectors/list", s.tornjakSelectorsList)
+	apiRtr.HandleFunc("/api/tornjak/v1/selectors/list", s.tornjakSelectorsList)
 	apiRtr.HandleFunc("/api/tornjak/agents/list", s.tornjakAgentsList)
+	apiRtr.HandleFunc("/api/tornjak/v1/agents/list", s.tornjakAgentsList)
 	// Clusters
 	apiRtr.HandleFunc("/api/tornjak/clusters/list", s.clusterList)
+	apiRtr.HandleFunc("/api/tornjak/v1/clusters/list", s.clusterList)
 	apiRtr.HandleFunc("/api/tornjak/clusters/create", s.clusterCreate)
+	apiRtr.HandleFunc("/api/tornjak/v1/clusters/create", s.clusterCreate)
 	apiRtr.HandleFunc("/api/tornjak/clusters/edit", s.clusterEdit)
+	apiRtr.HandleFunc("/api/tornjak/v1/clusters/edit", s.clusterEdit)
 	apiRtr.HandleFunc("/api/tornjak/clusters/delete", s.clusterDelete)
+	apiRtr.HandleFunc("/api/tornjak/v1/clusters/delete", s.clusterDelete)
```

Comment:
Thanks for your contribution! Minor comments:
- Can we move each of these new APIs down to be one block below the old API? It will be easier for code-readability to track these new APIs in one place
- Also please switch `/api/tornjak/v1/...` to `/api/v1/tornjak/...`